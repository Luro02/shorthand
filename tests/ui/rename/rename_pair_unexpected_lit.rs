use shorthand::ShortHand;

#[derive(ShortHand)]
struct Command {
    #[shorthand(rename(format = b"binary"))]
    first: usize,
}

fn main() {}
