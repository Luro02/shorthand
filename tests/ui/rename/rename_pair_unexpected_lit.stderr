error: unexpected literal type `byte string`
 --> $DIR/rename_pair_unexpected_lit.rs:5:33
  |
5 |     #[shorthand(rename(format = b"binary"))]
  |                                 ^^^^^^^^^
